docs/docs/user-guide.md

```diff
@@ -121,47 +148,56 @@ class Example {
 
 ## `@NullMarked`
```

Comment:
and `@NullUnmarked`, perhaps

Reply:
I guess. I was hoping not to emphasize it, since we expect its usage to be rare, and it doesn't serve a useful purpose in the model.